src/controls/Controls.spec.js

```diff
@@ -1 +1,69 @@
-// Test away!
\ No newline at end of file
+// Test away!
+import React from "react";
+import renderer from "react-test-renderer"; // 1: install this npm module as a dev dependency
+import { render, fireEvent } from "@testing-library/react";
+
+import Controls from "./Controls";
+
+describe("<Controls />", () => {
+    // 2. write this test
+    it("matches snapshot", () => {
+      const tree = renderer.create(<Controls />); // generates a DOM tree
+  
+      // snapshots are a JSON representation of the DOM tree
+      expect(tree.toJSON()).toMatchSnapshot();
+    });
+
+    // it("displays a message when 'speak' button is clicked", () => {
+    //     const tree = render(<Controls />);
+
+    //     tree.getByText(/Unlocked/i)
+    //     tree.getByText(/Locked/i)
+
+    //   });
+    it("should have close and lock by default", () =>{
+        const { getByText } = render(<Controls />);
+
+        expect(getByText(/close gate/i));
+        expect(getByText(/lock gate/i));
+    })
+    it("close gate", ()=>{
+        const { getByText, findAllByText } = render(<Controls />);       
+
+        fireEvent.click(getByText(/close gate/i));
+        expect(findAllByText(/lock gate/i));
```

Comment:
you were right all along findByText is the right way to search after you do the fireEvent, still don't know why you have to do All though I think you computer is being buggy